### DISCUSSION TITLE
change password pattern

### DISCUSSION THREAD

**Discussion by Maxpumpe** [pimcore/pimcore#8008] [Q&A / Support]
How can Change the password pattern in Pimcore?

**Comment by NiklasBr** [comment]
The default created passwords using the "flash" button or the password validation rule?

**Reply by Maxpumpe** [reply]
My aim is to be able to use simple passwords, such as 8 characters and a special character

**Reply by NiklasBr** [reply]
You can use 9 characters plus a special character, the only limitation is the minimum length. If you want something less secure you [should be able to override](https://symfony.com/doc/4.4/bundles/override.html) the `Route` at
https://github.com/pimcore/pimcore/blob/4fd1f8d1889b0c5e5f315b6aa1a6dad55fb2e362/bundles/AdminBundle/Controller/Admin/UserController.php#L269-L278

to implement your own Controller::action in your AppBundle, and maybe override the JS helper as well at https://github.com/pimcore/pimcore/blob/f7717e30031b8d9fe76cc9233dd0093024b7951e/bundles/AdminBundle/Resources/public/js/pimcore/helpers.js#L2692-L2715
